docs/sdk/migrations/6.0.0.md

```diff
@@ -0,0 +1,491 @@
+
+const encryptedData = await LitJsSdk.encryptFileAndZipWithMetadata({
+  sessionSigs,
+	...
+});
+```
+
+##### decryptZipFileWithMetadata
+
+- v5 - optional `authSig` or `sessionSigs`, but either must exist.
+- v6 - `sessionSigs` is strictly required
+
+```tsx
+// v5
+import * as LitJsSdk from '@lit-protocol/lit-node-client';
+import { AuthSig, SessionSigsMap } from '@lit-protocol/types';
+
+const authSig: AuthSig = '...';
+const sessionSigs: SessionSigsMap = '...';
+
+const decryptedData = await LitJsSdk.decryptZipFileWithMetadata({
+  authSig?: authSig,
+	sessionSigs?: sessionSigs;
+	...
+});
+  
+// v6
+import * as LitJsSdk from '@lit-protocol/lit-node-client';
+import { SessionSigsMap } from '@lit-protocol/types';
+
+const sessionSigs: SessionSigsMap = '...';
+
+const decryptedData = await LitJsSdk.decryptZipFileWithMetadata({
+  sessionSigs,
+	...
+});
+```
+
+#### `@lit-protocol/constants`
+
+##### Getting Lit supported curves
+
+`SIGTYPE` to `LIT_CURVE`
+
+```tsx
+// v5
+import { SIGTYPE } from '@lit-protocol/constants'
+
+export enum SIGTYPE {
+  BLS = 'BLS',
+  EcdsaK256 = 'K256',
+  EcdsaCaitSith = 'ECDSA_CAIT_SITH', 
+  EcdsaCAITSITHP256 = 'EcdsaCaitSithP256',
+}
+
+// v6
+import { LIT_CURVE } from '@lit-protocol/constants'
+
+export enum LIT_CURVE {
+  BLS = 'BLS',
+  EcdsaK256 = 'K256',
+  EcdsaCaitSith = 'ECDSA_CAIT_SITH',
+  EcdsaCAITSITHP256 = 'EcdsaCaitSithP256',
+}
+```
+## Enhancements, Fixes, and Additions
+
+#### signSessionKey
+
+In the `signSessionKey` parameters, we've added three extra optional arguments for [custom authentication](../wallets/auth-methods/custom-auth.md). These arguments allow you to use your PKP to [conditionally sign](../serverless-signing/conditional-signing.md) (via custom Lit Action code) the session key on the Lit nodes. For context, this function is typically used as the callback function for the `authNeededCallback` parameter of `getSessionSigs`.
+
+For simplicity, we introduced `getPkpSessionSigs`, which uses `signSessionKey` under the hood.
+
+```tsx
+// v5
+const response = await this.signSessionKey({
+  sessionKey: props.sessionKey,
+  statement: props.statement || 'Some custom statement.',
+  authMethods: [...params.authMethods],
+  pkpPublicKey: params.pkpPublicKey,
+  expiration: props.expiration,
+  resources: props.resources,
+  chainId: 1,
+  resourceAbilityRequests: props.resourceAbilityRequests,
+});
+
+// v6
+const response = await this.signSessionKey({
+  sessionKey: props.sessionKey,
+  statement: props.statement || 'Some custom statement.',
+  authMethods: [...params.authMethods],
+  pkpPublicKey: params.pkpPublicKey,
+  expiration: props.expiration,
+  resources: props.resources,
+  chainId: 1,
+  resourceAbilityRequests: props.resourceAbilityRequests,
+
+  // -- optional fields
+  ...(props.litActionCode && { litActionCode: props.litActionCode }),
+  ...(props.ipfsId && { ipfsId: props.ipfsId }),
+  ...(props.jsParams && { jsParams: props.jsParams }),
```

Comment:
Do we want to mention what `props` are? Since they are provided from the `authNeededCallback` perhaps we show that as well?

Reply:
@a1ttech here's the `AuthCalbackParams` interface for the props  https://github.com/LIT-Protocol/js-sdk/blob/635ec8da948103f3ce14271ccfc00393bc9e02a2/packages/types/src/lib/interfaces.ts#L45